pipelines/format.nox.py

```diff
@@ -43,7 +43,7 @@ def reformat_code(session: nox.Session) -> None:
     session.run("black", *config.PYTHON_REFORMATTING_PATHS)
 
 
-@nox.session(default_session=True, venv_backend="none")
+@nox.session(venv_backend="none")
 def check_trailing_whitespaces(session: nox.Session) -> None:
```

Comment:
Doesn't really make sense to run this by default when `reformat-code` is already calling the same logic just in write mode